### DISCUSSION TITLE
Stop Stenographer from capturing SSL pcaps

### DISCUSSION THREAD

**Discussion by Wilk4013** [Security-Onion-Solutions/securityonion#8737] [Unsupported Versions] [answered]
Hello,
I recall seeing a way for me to stop SSL sessions from Stenographer packet capture. Can someone point me in the right direction?
Thanks

**Comment by presianbg** [comment, marked as answer]
The most effective way is to implement a BPF for commonly encrypted traffic like HTTPS (443/tcp):
https://docs.securityonion.net/en/2.3/bpf.html?highlight=BPF#global-bpf

If you are not familiar with BPF, my recommendation is to start from here:
https://www.youtube.com/watch?v=uamNOjtUi4Y

Cheers!

**Reply by xfaith** [reply]
Just to chime in also directly from the docs:

steno:
  bpf:
    - not port 443

**Reply by Wilk4013** [reply]
Thanks to you both. Just what I needed
